config/reader.go

```diff
@@ -739,7 +708,7 @@ func CreateNewGRPCClient(ctx context.Context, cloudCfg *Cloud, logger logging.Lo
 		dialOpts = append(dialOpts, rpc.WithInsecure())
 	}
 
-	return rpc.DialDirectGRPC(ctx, u.Host, logging.GetOrNewLogger("rdk.networking"), dialOpts...)
```

Comment:
`GetOrNewLogger` is gone. In this case, it would have gotten a logger named `rdk.networking` from the registry and created it if it didn't exist. That logic is now handled automatically by `Sublogger`.